pkg/sbproxy/notifications/handlers/visibility_handler_test.go

```diff
@@ -278,6 +283,7 @@ var _ = Describe("Visibility Handler", func() {
 			Context("when an error occurs while enabling access", func() {
 				BeforeEach(func() {
 					fakeVisibilityClient.EnableAccessForPlanReturns(fmt.Errorf("error"))
+					fakeBrokerClient.GetBrokerByNameReturns(nil, nil)
```

Comment:
maybe add a test for new behavior you can achieve it by writing test with this setup:
 fakeVisibilityClient.EnableAccessForPlanReturnsOnCall(0, fmt.Errorf("error"))
fakeVisibilityClient.EnableAccessForPlanReturnsOnCall(1, nil)
fakeBrokerClient.GetBrokerByNameReturns(nil, nil)